packages/web3/test/stress/manual/browser_test/connection.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```

Comment:
if its static html for manual testing, add it in script commands in package json and use some tiny http server https://github.com/web3/web3.js/pull/6595/files#r1398936802, or
why cant cypress be used for it?

Reply:
no reason to use http server as we are connecting to an infura endpoint
as well discussed this in chat, i found it easier to use html than putting an extremely long test in jest/cypress. but it could use investigations with proper framework for long test.